recipes/dask-geopandas/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "0.1.0a2" %}
```

Comment:
This says it is an alpha release. Do we have a final release?

Reply:
It's a first ever release, and I think the idea for now was to keep doing alpha releases for a while (but already make it easier for people to install it with conda-forge). 
But if conda-forge prefers non-pre-release numbers, we can consider changing the version number, doesn't matter too much.

Reply:
That should be fine. We have [this wording in CFEP 5]( https://github.com/conda-forge/cfep/blob/master/cfep-05.md#exceptions ), which should allow for this

Reply:
Indeed, there has not yet been a non-alpha release before, so we should fall under this exception rule